src/Model/Export/Catalog/ProductField/GenericField.php

```diff
@@ -12,8 +12,11 @@
 use Omikron\Factfinder\Model\Export\Catalog\AttributeValuesExtractor;
 use Magento\Catalog\Model\Product;
 
+#[\AllowDynamicProperties]
```

Comment:
This is not necessary, since you now defined the `$attribute` member variable below (via my previous PR).